src/main/java/uk/gov/cabinetoffice/csl/domain/learnerrecord/actions/ICourseRecordAction.java

```diff
@@ -5,11 +5,12 @@
 import uk.gov.cabinetoffice.csl.service.messaging.model.IMessageMetadata;
 import uk.gov.cabinetoffice.csl.service.notification.messages.IEmail;
 
+import java.time.LocalDateTime;
 import java.util.Collection;
 
 public interface ICourseRecordAction {
 
-    CourseRecord applyUpdatesToCourseRecord(CourseRecord courseRecord);
+    CourseRecord applyUpdatesToCourseRecord(CourseRecord courseRecord, LocalDateTime completionDate);
```

Comment:
This shouldn't be part of the interface if it only affects 1 of the actions. The constructor for `RollupCompleteModule.java` should be used for contextual data like this.